packages/wallet/src/KeyManagement/util/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum DerivationPath {
```

Comment:
How about:
```suggestion
export enum CardanoKeyConst {
```

Reply:
Amazing!!! I was trying to figure out a better name for this. Thanks @mkazlauskas

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/251#discussion_r882355021